llvm/lib/Target/AMDGPU/AsmParser/AMDGPUAsmParser.cpp

```diff
@@ -6066,6 +6066,12 @@ bool AMDGPUAsmParser::ParseDirectiveAMDHSAKernel() {
                        ExprVal, ValRange);
       if (Val)
         ImpliedUserSGPRCount += 1;
+    } else if (ID == ".amdhsa_uses_cu_stores") {
+      if (!isGFX1250())
```

Comment:
If this is supposed to be a software controlled setting, it probably should be a separate attribute